Core/gb_struct_def.h

```diff
@@ -1,5 +1,12 @@
 #ifndef gb_struct_def_h
 #define gb_struct_def_h
+#include <stdint.h>
 struct GB_gameboy_s;
 typedef struct GB_gameboy_s GB_gameboy_t;
+
+#ifdef GB_16BIT_OUTPUT_COLOR
```

Comment:
I'm not sure if new `gc_struct_def.h` is a good location for this define, but it seems that it is only working option.